For some reason, anaconda can't resolve dependency on crc8 module during
installation. Luckily, it's quite small, so we can just bundle it
for the time being.
Index: src/drivers/net/ethernet/qlogic/qed/Makefile
===================================================================
--- src.orig/drivers/net/ethernet/qlogic/qed/Makefile	2018-08-20 06:03:02.460102562 +0200
+++ src/drivers/net/ethernet/qlogic/qed/Makefile	2018-08-20 10:33:58.172193281 +0200
@@ -9,3 +9,5 @@
 qed-$(CONFIG_QED_ISCSI) += qed_iscsi.o
 qed-$(CONFIG_QED_FCOE) += qed_fcoe.o
 qed-$(CONFIG_QED_OOO) += qed_ooo.o
+
+qed-y += crc8.o
Index: src/drivers/net/ethernet/qlogic/qed/crc8.c
===================================================================
--- src.orig/drivers/net/ethernet/qlogic/qed/crc8.c	2018-08-20 10:30:26.699219902 +0200
+++ src/drivers/net/ethernet/qlogic/qed/crc8.c	2018-08-20 10:36:50.284543862 +0200
@@ -14,7 +14,7 @@
  * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#define pr_fmt(fmt)		KBUILD_MODNAME ": " fmt
+#define pr_fmt(fmt)		"crc8" ": " fmt
 
 #include <linux/module.h>
 #include <linux/crc8.h>
@@ -40,7 +40,6 @@
 			table[i+j] = table[j] ^ t;
 	}
 }
-EXPORT_SYMBOL(crc8_populate_msb);
 
 /*
  * crc8_populate_lsb - fill crc table for given polynomial in regular bit order.
@@ -61,7 +60,6 @@
 			table[i+j] = table[j] ^ t;
 	}
 }
-EXPORT_SYMBOL(crc8_populate_lsb);
 
 /*
  * crc8 - calculate a crc8 over the given input data.
@@ -79,8 +77,7 @@
 
 	return crc;
 }
-EXPORT_SYMBOL(crc8);
 
-MODULE_DESCRIPTION("CRC8 (by Williams, Ross N.) function");
-MODULE_AUTHOR("Broadcom Corporation");
-MODULE_LICENSE("Dual BSD/GPL");
+/* MODULE_DESCRIPTION("CRC8 (by Williams, Ross N.) function"); */
+/* MODULE_AUTHOR("Broadcom Corporation"); */
+/* MODULE_LICENSE("Dual BSD/GPL"); */
